### DISCUSSION TITLE
BinlogLoad方式，FE提交数据到BE后，数据没有写入

### DISCUSSION THREAD

**Discussion by alunshqu** [apache/doris#17498] [RFC]
使用binlogLoad方式监听mysql数据导入doris，一开始写入几条数据后夯住，任务没报错，但是就是数据没进来，fe里边的日志也看起来是正常的，执行submit也OK，但就是数据无法看到

**Comment by yagagagaga** [comment]
binlog load 稳定性较差，目前没有维护了，后续准备删除掉这个功能，建议改用 FlinkCDC 的方式导入数据。

Binlog load has poor stability and is not maintained at present. It is planned to delete this feature later. It is recommended to import data using FlinkCDC.
